atomsci/ddm/utils/datastore_functions.py

```diff
@@ -1871,13 +1878,13 @@ def search_files_interactive (bucket='all', client=None, to_return='df', display
             new_value = new_value.replace("]","")
             new_value = new_value.split(",")
             new_value = [x.strip(' ') for x in new_value]
-            if type(values_for_key) == np.ndarray:
+            if type(values_for_key) is np.ndarray:
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.